### PR TITLE
i18n(zh-cn): translate `guide/integrations-guide/netlify.mdx`

### PR DESCRIPTION
**Pull request by leohxj**
#### What kind of changes does this PR include?

- Translated content

#### Description

- translate `guide/integrations-guide/netlify.mdx`

### CODE REVIEW

**Comment by leohxj** [comment]
cc @liruifengv

File: src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+---
+
+import Video from '~/components/Video.astro';
+import DontEditWarning from '~/components/DontEditWarning.astro';
+
+<DontEditWarning/>
+
+此适配器允许 Astro 部署 SSR 网站到 [Netlify](https://www.netlify.com/)。
+
+学习如何在我们的 [Netlify 部署指南](/en/guides/deploy/netlify/) 上部署你的 Astro 网站.
```
Comment (Yan-Thomas) [inline src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx:18]: fix link
```suggestion
学习如何在我们的 [Netlify 部署指南](/zh-cn/guides/deploy/netlify/) 上部署你的 Astro 网站.
```

File: src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+    excludeMiddleware: true,
+  },
+});
+```
+
+#### 传递 edge 上下文到你的站点
+
+Netlify 的 Edge Functions 提供了一个 [context 对象](https://docs.netlify.com/edge-functions/api/#netlify-specific-context-object)，其中包含有关请求的元数据，例如用户的 IP、地理位置数据和 cookie。
+
+为了将这个上下文中的值暴露给你的站点，在你项目的 [srcDir](/zh-cn/reference/configuration-reference/#srcdir) 中创建一个 `netlify-edge-middleware.ts`（或 `.js`）文件。这个文件必须导出一个函数，它返回要添加到 [Astro 的 `locals` 对象](/en/guides/middleware/#locals)中的数据，该对象在中间件和 Astro 路由中可用。
```
Comment (Yan-Thomas) [inline src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx:103]: fix link
```suggestion
为了将这个上下文中的值暴露给你的站点，在你项目的 [srcDir](/zh-cn/reference/configuration-reference/#srcdir) 中创建一个 `netlify-edge-middleware.ts`（或 `.js`）文件。这个文件必须导出一个函数，它返回要添加到 [Astro 的 `locals` 对象](/zh-cn/guides/middleware/#locals)中的数据，该对象在中间件和 Astro 路由中可用。
```

File: src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+
+## 参与贡献
+
+这个项目由 Astro 核心团队维护。欢迎提交问题或 PR！
+
+## 更新日志
+
+参加 [CHANGELOG.md](https://github.com/withastro/astro/tree/main/packages/integrations/netlify/CHANGELOG.md), 了解本集成的历史变更。
+
+[astro-integration]: /en/guides/integrations-guide/
```
Comment (Yan-Thomas) [inline src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx:277]: fix link
```suggestion
[astro-integration]: /zh-cn/guides/integrations-guide/
```

File: src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+
+学习如何在我们的 [Netlify 部署指南](/zh-cn/guides/deploy/netlify/) 上部署你的 Astro 网站.
+
+## 为什么是 Astro Netlify
+
+如果你正在使用 Astro 作为静态站点生成器——也就是它的开箱即用的行为——你不需要一个适配器。
+
+如果你希望 [使用服务器端渲染（SSR）](/zh-cn/guides/server-side-rendering/)，Astro 需要一个与你的部署运行时相匹配的适配器。
+
+[Netlify](https://www.netlify.com/) 是一个部署平台，允许您通过直接连接 GitHub 仓库来托管您的网站。这个适配器增强了 Astro 的构建流程，为通过 Netlify 部署项目做好准备。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx:26]:
```suggestion
[Netlify](https://www.netlify.com/) 是一个部署平台，允许你通过直接连接 GitHub 仓库来托管您的网站。这个适配器增强了 Astro 的构建流程，为通过 Netlify 部署项目做好准备。
```

File: src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+
+    export default defineConfig({
+      output: 'server',
+      adapter: netlify(),
+    });
+    ```
+
+### Edge Functions
+
+Netlify 有两个无服务器平台，[Netlify Functions](https://docs.netlify.com/functions/overview/) 和 [Netlify Edge Functions](https://docs.netlify.com/edge-functions/overview/)。使用 Edge Functions，您的代码分布在更接近用户的地方，降低延迟。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx:64]:
```suggestion
Netlify 有两个 serverless 平台，[Netlify Functions](https://docs.netlify.com/functions/overview/) 和 [Netlify Edge Functions](https://docs.netlify.com/edge-functions/overview/)。使用 Edge Functions，你的代码分布在更接近用户的地方，降低延迟。
```

File: src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+    },
+  });
+}
+```
+
+## 示例
+
+* [Astro Netlify Edge Starter](https://github.com/sarahetter/astro-netlify-edge-starter) 提供了一个示例以及 README 中的指南。
+
+* [Browse Astro Netlify projects on GitHub](https://github.com/search?q=%22%40astrojs%2Fnetlify%22+filename%3Apackage.json\&type=Code) 提供更多示例!
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx:261]: @Yan-Thomas Here seems to be an incorrect link.

**Review by liruifengv — CHANGES_REQUESTED** [review]

File: src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx (hunk 6 above)
Comment (leohxj) [reply src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx:261]: incrrect link, it maybe `https://github.com/search?q=%22%40astrojs%2Fnetlify%22+filename%3Apackage.json&type=repositories`, type should be repositories?

Comment (leohxj) [reply src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx:261]: @Yan-Thomas how is going on?

Comment (Yan-Thomas) [reply src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx:261]: We have made an PR to update the link, see:
https://github.com/withastro/astro/pull/7862

After it's merged I'll come here, update this and merge!

Comment (leohxj) [reply src/content/docs/zh-cn/guides/integrations-guide/netlify.mdx]: update the link.

**Pull request merged**